### ISSUE TITLE
Unable to get bounding box that tightly encloses the object

### ISSUE THREAD

**Issue by prantiksen4** [experiencor/basic-yolo-keras#190]
Hi, I tried training the full yolo implementation using my own data (I have a good amount of data). Turned out that the bounding box though are decent but they don't tightly enclose the desired object. Any pointers to this will be of great help. My training data has objects at various scales.

**Comment by experiencor** [comment]
@prantiksen4 You may try to increase coord_scale for tighter box predictions.

**Comment by pratt-datar** [comment]
should we train it again?

**Comment by experiencor** [comment]
@pratt-datar I make the new repo (YOLOv3). The new loss function has a better way to penalize box size's loss. You may check it out. (https://github.com/experiencor/keras-yolo3)

P/S: it's a work in progress so expect to spot a lot of bugs.
